nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/java/org/apache/nifi/processors/azure/storage/ListAzureDataLakeStorage.java

```diff
@@ -102,6 +104,7 @@
         "where the previous node left off, without duplicating the data.")
 public class ListAzureDataLakeStorage extends AbstractListProcessor<ADLSFileInfo> {
 
+    public static final String TEMP_FILE_PREFIX = "nifitemp_";
```

Comment:
Although this variable is `public`, it does not appear to be referenced outside of the processor, and `PutAzureDataLakeStorage` has the same value.  It seems like this value should be abstracted to a shared location.